test/bugs-reported/bug1.chpl

```diff
@@ -0,0 +1,100 @@
+
```

Comment:
Could we shorten this program down to something simpler? I think this could be done in a couple of lines:

```chpl
var x = 5;
writeln(x++);
```